src/main/java/com/github/manolo8/darkbot/core/manager/FacadeManager.java

```diff
@@ -123,13 +126,15 @@ public void install(BotInstaller botInstaller) {
 
     public void tick() {
         // Currently commands are not used by the bot and they represent
-        // a decently big cpu chunk in ticking. Leaving them out reduces tick time significantly.
-        //commands.update();
+        // a decently big cpu chunk in ticking. Leaving them out reduces tick time
+        // significantly.
+        // commands.update();
```

Comment:
if you're going to reformat it try to do so by moving the a upwards see if it can fit in 2 lines instead of 3 with a single word leftover

Reply:
Fixed